packages/clay-core/src/vertical-bar/VerticalBar.tsx

```diff
@@ -58,6 +58,16 @@ type Props = {
 	 * Callback is called when the active state changes (controlled).
 	 */
 	onActiveChange?: InternalDispatch<React.Key | null>;
+
+	/**
+	 * Sets a custom width on the sidebar panel.
+	 */
+	panelWidth?: number | undefined;
```

Comment:
I'm not sure if it would be a good idea to provide an API to set the width, I think it would be better to provide an API to set the maximum size instead which would be used for the clamp.

Reply:
Hmm I don't know if I understood correctly, what would be the use case for setting the `panelWidth`? I think we still need to offer the `panelWidthMax` to use in the clamp.

Reply:
Yes I'm including `panelWidthMin` and `panelWidthMax`. The use case for panelWidth is remembering the `panelWidth` via `SessionClicks.get(request, "id", "defaultValue")` in jsps.

Reply:
I think the component can handle this internally by saving to local storage.

Reply:
We might run into a problem by handling this internally. The `id` isn't guaranteed to stay the same. We would lose reference to it.